scripts/page_browse.js

```diff
@@ -145,7 +151,7 @@ function makeTextWidget(container, splitter = false, reLayout = null) {
 var viewSplitter = function(container, storageKey) {
     const storageKeyLayout = storageKey + "-layout";
     let layout = JSON.parse(localStorage.getItem(storageKeyLayout));
-    if(!$.isPlainObject(layout)) {
+    if(!layout || layout.splitDirection !== "horizontal" || layout.splitDirection !== "vertical") {
```

Comment:
Should be
`if(!layout || (layout.splitDirection !== "horizontal" && layout.splitDirection !== "vertical")) {`
as you have elsewhere.